tests/listsAndDropDowns.spec.ts

```diff
@@ -1,69 +1,59 @@
 import { test, expect } from '@playwright/test';
 
 test.describe('Lists and Drop Downs', () => {
+    //declare variables
+    let petTypeInputField
+    let petNameInputField
+
     test.beforeEach(async ({ page }) => {
         await page.goto('/')
         await page.getByRole('button', { name: 'Owners' }).click()
         await page.getByRole('link', { name: 'Search' }).click()
-        await expect(page.getByRole('button', { name: 'Owners' })).toHaveText('Owners');
+        const ownersButton = page.getByRole('button', { name: 'Owners' })
+        await expect(ownersButton).toContainText('Owners')
+        //initilize reusable locators
+        petTypeInputField = page.locator('#type')
+        petNameInputField = page.locator('#name')
     })
 
     test('Test Case 1: Validate selected pet types from the list', async ({ page }) => {
         await page.getByRole('link', { name: 'George Franklin' }).click()
         await expect(page.locator('.ownerFullName')).toHaveText('George Franklin')
         await page.getByRole('button', { name: 'Edit Pet' }).click()
-        await expect(page.locator('#name')).toHaveValue('Leo')
+        await expect(petNameInputField).toHaveValue('Leo')
         await expect(page.locator('#owner_name')).toHaveValue('George Franklin')
-        await expect(page.locator('#type')).toHaveValue('cat')
-        const dropdownMenu = page.locator('#type')
+        await expect(petTypeInputField).toHaveValue('cat')
+        const dropdownMenu = petTypeInputField
         await dropdownMenu.click()
         const typesOfPets = ['cat', 'dog', 'lizard', 'snake', 'bird', 'hamster']
 
         for (const pet of typesOfPets) {
             await dropdownMenu.selectOption(pet)
             await expect(dropdownMenu).toHaveValue(pet)
-            await expect(page.locator('#type')).toHaveValue(pet)
+            await expect(petTypeInputField).toHaveValue(pet)
             await dropdownMenu.click()
         }
     })
 
     test('Test Case 2: Validate the pet type update', async ({ page }) => {
         await page.getByRole('link', { name: 'Eduardo Rodriquez' }).click()
-        await page.getByRole('button', { name: 'Edit Pet' }).nth(1).click()
-        const typeOfPet = page.locator('#type')
-        const nameOfPet = page.locator('#name')
-        await expect(nameOfPet).toHaveValue('Rosy')
-        await expect(typeOfPet).toHaveValue('dog')
-        const dropdownMenu = page.locator('#type')
-        await dropdownMenu.click()
-        const typesOfPets = ['cat', 'dog', 'lizard', 'snake', 'bird', 'hamster']
+        const EditButton = page.locator('.dl-horizontal', { hasText: "Rosy" }).getByRole('button', { name: 'Edit Pet' })
+        await EditButton.click()
```

Comment:
Constants always should start with lower case. Capital case reserved only for the class names and constructor of the functions in TypeScript